app/src/main/java/com/example/demoquizwithandroid/QuizViewModel.kt

```diff
@@ -8,38 +8,38 @@ import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import io.appwrite.Client
 import io.appwrite.exceptions.AppwriteException
-import io.appwrite.services.Database
+import io.appwrite.services.Databases
 import kotlinx.coroutines.launch
-import org.json.JSONArray
-import org.json.JSONObject
 
 class QuizViewModel : ViewModel() {
-    private val collectionId = "606d5bd9626ae"
+    private val url = App.appResources!!.getString(R.string.URL)
+    private val projectId = App.appResources!!.getString(R.string.PROJECT_ID)
+    private val databaseId = App.appResources!!.getString(R.string.DATABASE_ID)
+    private val collectionId = App.appResources!!.getString(R.string.COLLECTION_ID)
```

Comment:
Let's add these in code as constants so we don't need to access resources from the ViewModel